### PR TITLE
Improvements in Active VIsits Widget

### PR DESCRIPTION
**Pull request by vasharma05**
### What does this PR include?

This PR is in continuation to #15 with some improvements mentioned below.
1. Visual Improvement
2. Search added
3. Empty state added

image
image
image

### CODE REVIEW

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx [hunk 1]
```diff
@@ -83,56 +90,69 @@ const ActiveVisitsTable = (props) => {
...
     });
     return () => activeVisits.unsubscribe();
   }, []);
-  return (
+
+  return !loading ? (
     <div className={styles.activeVisitsContainer}>
       <div className={styles.activeVisitsDetailHeaderContainer}>
-        <h4 className={styles.productiveHeading02}>{t('activeVisits', 'Active Visits in Clinic')}</h4>
+        <h4 className={styles.productiveHeading02}>{t('activeVisits', 'Active Visits')}</h4>
```
Comment (FlorianRappl) [inline packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx:104]: Changed here but not in the actual language file.

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 .activeVisitsContainer {
   background-color: $ui-background;
-  width: 100%;
+  // feedback needed
```
Comment (FlorianRappl) [inline packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss:5]: Yes. Why 90? Why `vw`?

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx (hunk 1 above)
Comment (vasharma05) [reply packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx:104]: Ohh yeah, Thanks!

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss (hunk 2 above)
Comment (vasharma05) [reply packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss:5]: If you see the current widget on the home screen, the width changes according to the content in different pages and hence I wanted to fix the size. If I fix the size to 768px(tab width) then the date divided into 2 lines. I discussed it with JJ as in to what to do, he said, you can help in it and thought of keeping to the screen width and hence `90vw` .

**Review by FlorianRappl — APPROVED** [review]

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss (hunk 2 above)
Comment (FlorianRappl) [reply packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss:5]: Yes, but 90vw does not make much sense, e.g., on mobile. You shouldn't fix the width and you should try to make a progressive design, i.e., one that pretty much *always* works.

So what if the nav bar is shown on the left side? What if the user has a wide screen (like 4k pixels wide)? 90vw does not sound right,.

Comment (vasharma05) [reply packages/esm-active-visits-app/src/active-visits-widget/active-visits.scss:5]: Sure, I get you, let me fix that too.
Thanks!

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**